packit_service/worker/handlers/testing_farm_handlers.py

```diff
@@ -178,14 +174,6 @@ def run(self) -> TaskResults:
             status = CommitStatus.failure
             summary = self.summary or "Tests failed ..."
 
-        if len(self.tests) > 0 and self.tests[0].name == "/packit/install-and-verify":
```

Comment:
aha, so the installability test has been the use case for the xunit parsing.
I see you solved it elsewhere so we should be good to go :-)